optuna_dashboard/_app.py

```diff
@@ -165,6 +165,7 @@ def rename_study(study_id: int) -> dict[str, Any]:
             response.status = 500
             return {"reason": "Failed to load the new study"}
 
+        note.copy_notes(storage, src_study, dst_study)
```

Comment:
Could you move this line to under the L155 to make it show error messages to users?